client/viame-web-common/components/AttributesPanel.vue

```diff
@@ -483,78 +464,75 @@ export default defineComponent({
           </div>
         </v-col>
       </v-row>
-      <v-subheader
+
+      <!-- DETECTION ATTRIBUTES -->
+      <div
         v-if="selectedDetection"
         class="border-highlight"
-        style="height:65px; min-height:65px"
       >
-        <v-container
-          dense
-          class="px-0 my-2"
+        <v-row
+          class="align-center"
+          no-gutters
```

Comment:
The only real change here is removing the `v-container`, which doesn't seem to be performing any function.